cmd/kubebuilder/util/stdin.go

```diff
@@ -18,21 +18,26 @@ package util
 
 import (
 	"bufio"
+	"fmt"
 	"log"
 	"os"
 	"strings"
 )
 
+var reader = bufio.NewReader(os.Stdin)
```

Comment:
This should be local to the `Yesno` function or taken as input in `Yesno`

Reply:
> local to the Yesno function

This won't work.

> taken as input in Yesno

This is viable.

Reply:
We can call `NewReader` inside api command. As long as we only create one reader, it will be ok.